--- xawtv/libng/plugins/snd-oss.c.orig	2011-04-02 09:59:50.000000000 +0200
+++ xawtv/libng/plugins/snd-oss.c	2011-04-02 10:00:09.000000000 +0200
@@ -342,7 +342,7 @@
     memset(h,0,sizeof(*h));
 
     if (-1 == (h->fd = open(device ? device : ng_dev.dsp,
-			    record ? O_RDONLY : O_WRONLY | O_NONBLOCK))) {
+			    record ? O_RDONLY : O_WRONLY | O_NONBLOCK, 0644))) {
 	fprintf(stderr,"oss: open %s: %s\n",
 		device ? device : ng_dev.dsp,
 		strerror(errno));
